packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -109,7 +110,10 @@ export class AtomicSegmentedFacet
   private renderValuesContainer(children: VNode[]) {
     const classes = 'box-container flex';
     return (
-      <FacetValuesGroup i18n={this.bindings.i18n} label={this.label}>
+      <FacetValuesGroup
+        i18n={this.bindings.i18n}
+        label={this.label !== undefined ? this.label : 'no-label'}
```

Comment:
The shorthand for to `this.label !== undefined ? this.label : 'no-label'` is `this.label ?? 'no-label'`.

However, I would make `label` optional in `FacetValuesGroup`